<ui version="4.0" >
 <class>TorrentOptsWidget</class>
 <widget class="QWidget" name="TorrentOptsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>437</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="labelStatus" >
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QToolBox" name="toolBox" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_3" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>355</width>
            <height>299</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Files to download</string>
          </attribute>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item row="0" column="0" >
            <widget class="QTreeWidget" name="treeFiles" >
             <property name="columnCount" >
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_6" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>355</width>
            <height>299</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Seeding limits</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QCheckBox" name="checkSeedRatio" >
             <property name="text" >
              <string>Seed up to ratio</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QDoubleSpinBox" name="doubleSeed" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="singleStep" >
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QCheckBox" name="checkSeedUpload" >
             <property name="text" >
              <string>Seed until specified amount
of data has been uploaded</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLineEdit" name="lineSeed" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>MB</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>81</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>108</width>
            <height>98</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>URL seeds</string>
          </attribute>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="2" >
            <widget class="QListWidget" name="listUrlSeeds" />
           </item>
           <item row="1" column="0" >
            <widget class="QLineEdit" name="lineUrl" />
           </item>
           <item row="1" column="1" >
            <widget class="QPushButton" name="pushAddUrlSeed" >
             <property name="text" >
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>170</width>
            <height>126</height>
           </rect>
          </property>
          <attribute name="label" >
           <string>Trackers</string>
          </attribute>
          <layout class="QGridLayout" >
           <property name="leftMargin" >
            <number>0</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>0</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="3" >
            <widget class="QListWidget" name="listTrackers" />
           </item>
           <item row="1" column="0" colspan="3" >
            <widget class="QLineEdit" name="lineTracker" />
           </item>
           <item row="2" column="0" >
            <widget class="QPushButton" name="pushTrackerAdd" >
             <property name="text" >
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="2" >
            <widget class="QPushButton" name="pushTrackerRemove" >
             <property name="text" >
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkSeedRatio</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSeed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>144</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel" >
     <x>269</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSeedUpload</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineSeed</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>89</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel" >
     <x>292</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
